src/components/ConferencePlanForm.tsx

```diff
@@ -272,12 +272,12 @@ class ConferencePlanForm extends React.Component<Props, State> {
    * @return {void}
    */
   handleSave = (): void => {
-    const feedback = this.state.feedback;
-    const questions = this.state.questions;
-    const addedQuestions = this.state.addedQuestions;
-    const notes = this.state.notes;
+    const feedbacks = this.state.feedback.filter(feedback => feedback !== '');
+    const questions = this.state.questions.filter(question => question !== '');
+    const addedQuestions = this.state.addedQuestions.filter(question => question !== '');;
+    const notes = this.state.notes.filter(note => note !== '');
```

Comment:
In situations like this I miss the Scala's `_` parameter :smile: We could do something like this with it
```
.filter(_ != '')
```
and go on with our lives :stuck_out_tongue: